tooling/reproducible/ReproducibleBuilds.md

```diff
@@ -79,17 +83,55 @@ before the comparable_patch.sh can be run.
 - For BinRepl.class : export CLASSPATH=<temurin-build>/tooling/src/java:$CLASSPATH
 - A JDK for running BinRepl java : export PATH=<jdk>/bin:$PATH
 
+##### Cygwin treacherousness
+
+- It is extremely difficult (maybe impossible) to invoke `vcvarsall.bat+cl` in cygwin directly
+- Thus, it is recommended to launch this via `cmd -c` or preferably by an executable `.bat` file such as:
+
+```bash
+ pushd "$MSVSC/BUILD/TOOLS"
+    rm -f WindowsUpdateVsVersionInfo.obj
+    echo "
+      call vcvars64.bat
+      cl $(cygpath -m $YOUR_WORKDIR/temurin-build/tooling/src/c/WindowsUpdateVsVersionInfo.c) version.lib
+    " > bat.bat
+    chmod 777 bat.bat
```

Comment:
using 777 is not a great example, is this actually needed?

Reply:
I was not able to get the proper results by executing "cmd bat.bat", thus downgraded to this. Probably 755 would be enough, but the bat.bat is/sohould be removed immediately.